blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md

```diff
@@ -0,0 +1,242 @@
+---
+title: How to Use OpenTelemetry to Monitor Next.js Applications
+createdAt: 2024-07-26T12:00:00Z
```

Comment:
@Vadman97 have any thoughts about how/when you want to roll these out?

Reply:
discussed offline, any dates work here